query_temp.py

```diff
@@ -1,186 +1,113 @@
 from sqlalchemy.sql import func
-from sqlalchemy import and_, or_, case
-
+from sqlalchemy.orm import aliased
+from sqlalchemy import and_, or_, case, distinct
+from sqlalchemy import (Table, Column, String, Integer,
+                        MetaData, select, func, join)
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from calendar import monthrange
 from pycds import Obs, History, Variable
 
 from query_helpers import total_days, total_years, count, min_time, max_time, days_in_month, hours_in_month
 
-def query_design_temp_percentile(start_time, end_time, session, quantile=0.01):
-    """A query to get the percentile of minimum daily air_temperatures
-    at a station in a given time frame. Daily minimum air
-    temperatures are used.
+def temp_quantile(
+    session, start_time, end_time,
+    method='lower', cell_method='time: minimum',
+    net_var_name='2', standard_name='air_temperature',
+    quantile=0.01, min_years=8
+    ):
+
+    '''A query to get the percentile of daily temperatures
+    at a station in a given time frame. Daily air
+    temperatures are used. Eligible data is used if a station
+    has min_years or more fully complete months of data.
     -----------------------------------------
     Args:
         session (sqlalchemy Session): session constructed using
             connection string and engine
-        percentile (float): desire percentile in fraction
-        month (datetime): desired month in which to calculate
-            the desired percentile, default is 1, or January.
+        method (str): calculate the upper or lower percentile
+        cell_method (str): type of cell method
```

Comment:
Be more explicit, in both parameter name and these comments, about what these parameters do, which is to select the variable of interest.

Reply:
I'm trying to follow the pycds conventions as best I can here. I've updated this docstring to include a link to pycds relevant documentation with a [full description of the parameter](https://data.pacificclimate.org/portal/docs/pycds.html).

Should I copy that information directly into my documentation as well?